pkg/apis/kops/componentconfig.go

```diff
@@ -609,6 +609,12 @@ type KubeSchedulerConfig struct {
 	// which has been supported as far back as Kubernetes 1.7. The default depends on the version and the cloud provider
 	// as outlined: https://kubernetes.io/docs/concepts/storage/storage-limits/
 	MaxPersistentVolumes *int32 `json:"maxPersistentVolumes,omitempty"`
+	// Qps sets the maximum qps to send to apiserver after the burst quota is exhausted
+	Qps *resource.Quantity `json:"qps,omitempty" configfile:"ClientConnection.QPS"`
+	// Burst sets the maximum qps to send to apiserver after the burst quota is exhausted
+	Burst *int32 `json:"burst,omitempty" configfile:"ClientConnection.Burst"`
```

Comment:
This is an `int32` in k/k, so why a pointer here?

Reply:
To allow leaving the field out of the config and let kube-scheduler just go with the default.

Reply:
If it's `int32` in k/k, the default is specified by a zero value. It is not necessary to make this a pointer to allow leaving the field out of the config.